code/modules/assembly/igniter.dm

```diff
@@ -1,3 +1,8 @@
+#define EXPOSED_VOLUME 1000
+#define ROOM_TEMP 293
+#define MIN_FREEZE_TEMP 50
+#define MAX_FREEZE_TEMP 1000000

```

Comment:
Undef these at the end of the file if they are not used elsewhere.

And if they are used elsewhere, place them on the defines folder.